I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script_ch10.rpy", line 94, in script
    jump chapter_11
ScriptError: could not find label 'chapter_11'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script_ch10.rpy", line 94, in script
    jump chapter_11
  File "C:\Software\RenPy\renpy-7.3.5-sdk\renpy\ast.py", line 1697, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Software\RenPy\renpy-7.3.5-sdk\renpy\script.py", line 898, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'chapter_11'.

Windows-8-6.2.9200
Ren'Py 7.3.5.606
Limbus inc. 1.0
Mon Aug 10 10:12:48 2020
